support-frontend/assets/helpers/redux/checkout/product/selectors/selectedAmount.ts

```diff
@@ -15,6 +16,18 @@ export function getUserSelectedAmount(state: ContributionsState): number {
 	return priceCardAmountSelected;
 }
 
+export function getAmountCoveringTransactionCost(
```

Comment:
A small nit in regards to readability - would this be clearer as `getAmountWithCoveringTransactionCost`?

Reply:
How about `getAmountWithTransactionCostCovered` ?